pkg/tfbridge/provider_test.go

```diff
@@ -960,6 +960,8 @@ func TestCheckConfig(t *testing.T) {
 	// Secrets: it is important to handle secrets correctly since the results of CheckConfig are stored in the
 	// state. If secrets are not marked as such, they are stored as plaintext in the state.
 	t.Run("preserve_program_secrets", func(t *testing.T) {
+		t.Skip()
```

Comment:
This is important - I don't think the bridge needs to do this, since it responds to the engine with "acceptSecrets: false" and can rely on the engine's heuristic to take care of the secrets.